apps/test-suite/tests/HTML.js

```diff
@@ -77,7 +78,6 @@ const viewElements = {
   Main,
   Section,
   Nav,
-  BR,
```

Comment:
We may want to look again into https://github.com/expo/expo/pull/7395 which (I hope) provides even more test coverage to `BR`. Would you take another look at it, please, @EvanBacon? 🙂